assets/schema/ensemble_schema.json

```diff
@@ -2759,6 +2759,10 @@
           "type": "string",
           "description": "Hint text explaining your widget"
         },
+        "floatingLabelText": {
```

Comment:
Isn't this just the label. Why introduce another variable here?

Reply:
```
label: Email
labelStyle: {
   type: outside | inline
}
```